llama-index-core/llama_index/core/extractors/metadata_extractors.py

```diff
@@ -192,14 +205,10 @@ async def _aextract_keywords_from_node(self, node: BaseNode) -> Dict[str, str]:
         if self.is_text_node_only and not isinstance(node, TextNode):
             return {}
 
-        # TODO: figure out a good way to allow users to customize keyword template
```

Comment:
@logan-markewich: not sure if you left this TODO, but if you did flagging you here in case you want to take a look to see if the proposed way to allow for customized prompt template is sufficient.

Reply:
This makes sense to me